docs/openapi/components/schemas/common/AgricultureProduct.yml

```diff
@@ -40,7 +40,7 @@ properties:
     type: string
     $linkedData:
       term: plu
-      '@id': https://w3id.org/traceability#plu
+      '@id': https://schema.org/identifier
```

Comment:
I searched a bit for a better term and did not find one. 

https://www.pma.com/content/articles/2014/05/plu-codes <-- 500 error for me.

Reply:
maybe better: https://www.ifpsglobal.com/PLU-Codes